Description: To ensure our vendor.js is included in omni.jar, we
 need to ensure it is in $(DIST)/$(STAGEPATH) at the time that the
 omni.jar is packed. It seems that the only way to do this is to
 copy it in to $(DIST)/bin before running make install, and then adding
 the file to the package-manifest
Author: Chris Coulson <chris.coulson@canonical.com>
Forwarded: not-needed
--- a/browser/installer/package-manifest.in
+++ b/browser/installer/package-manifest.in
@@ -437,6 +437,8 @@
 #ifdef MOZ_SERVICES_SYNC
 @BINPATH@/@PREF_DIR@/services-sync.js
 #endif
+; Ubuntu preferences
+@BINPATH@/@PREF_DIR@/vendor.js
 @BINPATH@/greprefs.js
 @BINPATH@/defaults/autoconfig/platform.js
 @BINPATH@/defaults/autoconfig/prefcalls.js
